app/views/form/page.html.erb

```diff
@@ -42,10 +42,10 @@
       <div class="govuk-button-group">
         <% if !@page.id.include?("value_check") && if request.query_parameters["referrer"] != "check_answers" %>
           <%= f.govuk_submit "Save and continue" %>
-          <%= govuk_link_to "Skip for now", send(@lettings_log.form.next_page_redirect_path(@page, @lettings_log, current_user), @lettings_log) %>
+          <%= govuk_link_to "Skip for now", send(@log.form.next_page_redirect_path(@page, @log, current_user), @log) %>
           <% else %>
           <%= f.govuk_submit "Save changes" %>
-            <%= govuk_link_to "Cancel", "/logs/#{@lettings_log.id}/setup/check-answers" %>
+            <%= govuk_link_to "Cancel", "/logs/#{@log.id}/setup/check-answers" %>
```

Comment:
Can we use a route here?

Reply:
Sorted now, we have added a route as suggested.